Tests/CommandsTests/BuildCommandTests.swift

```diff
@@ -794,20 +890,22 @@ struct BuildCommandTestCases {
 
     @Test(
         .requireHostOS(.macOS),
-        arguments: [BuildSystemProvider.Kind.xcode],
+        arguments: getBuildData(for: [BuildSystemProvider.Kind.swiftbuild, .xcode]),
     )
     func buildSystemOverrides(
-        buildSystem: BuildSystemProvider.Kind,
+        data: BuildData,
```

Comment:
I don't think we should default to running all these tests in debug and release - the difference isn't relevant to many of them and it extends the runtime quite a bit

Reply:
https://github.com/swiftlang/swift-package-manager/issues/8984 could say otherwise :)